source/_integrations/brother.markdown

```diff
@@ -26,3 +26,11 @@ To add `Brother Printer` to your installation, go to **Configuration** >> **Inte
 Some very old Brother printers use different data format and these models are not supported. The integration will show information about that during configuration.
 
 </div>
+
+## Configuring the printer
+
+To set SNMP, navigate to the printer's web interface (for example: http://192.168.5.6) and turn on under Network / Protocol / SNMP.
+For some Brother devices, `SNMPv3 read-write access and v1/v2c read-only access` is the option required (under advanced settings).
+
+![SNMP settings on Brother Printer web interface](https://user-images.githubusercontent.com/68903245/103670460-e239ad80-4f79-11eb-84f0-dd52a2ae7971.png)
```

Comment:
Images need to be stored and hosted from our own website. If you add graphics, please add it to this repository.

Reply:
oh ok - rolling...